collated/main/nahtanojmil.md

```diff
@@ -0,0 +1,361 @@
+# nahtanojmil
+###### \java\seedu\address\logic\commands\RemarkCommand.java
+``` java
+/**
+ * Changes the remark of an existing person in the address book.
+ */
+public class RemarkCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "remark";
+    public static final String COMMAND_ALIAS = "rm";
+
+    public static final String MESSAGE_ADD_REMARK_SUCCESS = "Added remark to person: %1$s";
+    public static final String MESSAGE_DELETE_REMARK_SUCCESS = "Removed remark from person: %1$s";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " : edits the remarks of a person "
+            + "by the index from the last shown list. Old remarks will be overwritten by new "
+            + "ones entered.\n"
+            + "parameters: INDEX (positive integer)" + PREFIX_REMARK + "[REMARK]\n"
+            + "Example: " + COMMAND_WORD + "3 " + PREFIX_REMARK + "likes to eat!";
+
+    private Index index;
+    private Remark stringRemark;
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.